initialization/initialization_test.go

```diff
@@ -878,20 +877,7 @@ func initData(datadir string, bitsPerLabel uint) ([]byte, error) {
 	}
 	defer reader.Close()
 
-	gsReader := shared.NewGranSpecificReader(reader, bitsPerLabel)
-	writer := bytes.NewBuffer(nil)
-	for {
-		b, err := gsReader.ReadNext()
-		if err != nil {
-			if err == io.EOF {
-				break
-			}
-			return nil, err
-		}
-		writer.Write(b)
-	}
-
-	return writer.Bytes(), nil
+	return io.ReadAll(reader)
```

Comment:
It's still the same behaviour as before so I guess it's OK, but `initData` reads ALL the data from disk into a single `[]byte` and returns it.

I assume this will cause issues when `datadir` contains more than a few MiB of data... Since this is only used in tests it's probably OK though.